python/pyhail/dataset.py

```diff
@@ -1202,15 +1573,15 @@ def split_multi(self, propagate_gq=False, keep_star_alleles=False):
 
         ``split_multi`` adds the following annotations:
 
-         - **va.wasSplit** (*Boolean*) -- true if this variant was
-           originally multiallelic, otherwise false.
+        - **va.wasSplit** (*Boolean*) -- true if this variant was
```

Comment:
Are we removing block quotes for all annotations? I'm happy with that, but we need to tell the rest of the group.